### PR TITLE
EPMRPP-42185 || Initial status attribute impl

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java [hunk 1]
```diff
@@ -270,6 +273,21 @@ public OperationCompletionRS updateTestItem(ReportPortalUser.ProjectDetails proj
 				.collect(toList());
 	}
 
+	private void checkInitialStatusAttribute(TestItem testItem) {
+		Optional<ItemAttribute> statusAttirbute = testItem.getAttributes()
+				.stream()
+				.filter(attribute -> attribute.getKey().equalsIgnoreCase(INITIAL_STATUS_ATTRIBUTE_KEY) && attribute.isSystem())
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java]: possible npe if attribute key is null
CONSTANT.equals(variable)

File: src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java [hunk 2]
```diff
@@ -221,6 +223,7 @@ public OperationCompletionRS updateTestItem(ReportPortalUser.ProjectDetails proj
 					INCORRECT_REQUEST,
 					"Unable to change status on test item with children"
 			);
+			checkInitialStatusAttribute(testItem);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java]: what about updating some tests according to the new logic?

File: src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java]: fixed

File: src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java (hunk 2 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java]: added

**Pull request merged**